src/components/Check/Check.scss

```diff
@@ -10,25 +10,37 @@
 $checkBoxHeight: 20px;
 
 .ms-Check {
-  display: inline-block;
-  cursor: default;
-  vertical-align: top;
-  width: $checkBoxHeight;
-  height: $checkBoxHeight;
+  font-family: FabricMDL2Icons;
+  line-height: 1;
   text-align: center;
+  width: 18px;
+  height: 18px;
+  vertical-align: top;
   position: relative;
+  cursor: default;
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
```

Comment:
you don't need every override; we have postcss autoprefixer for that. just use "user-select".